vexflow/vexflow.d.ts

```diff
@@ -6,7 +6,7 @@
 //inconsistent namespace: this is a helper funtion from tables.js and should not pollute the global namespace!
 declare function sanitizeDuration(duration : string) : string;
 
-declare namespace Vex {
+declare module Vex {
```

Comment:
I think it needs to be declared as a `module` for jspm to load it. I went off the lodash package as a reference.